{{cookiecutter.project_name}}/notebooks/Train.py

```diff
@@ -58,9 +58,9 @@
 
 # COMMAND ----------
 
-from mlflow.pipelines import Pipeline
+from mlflow.recipes import Recipe
 
-p = Pipeline(profile=profile)
+p = Recipe(profile=profile)
```

Comment:
Can we change this to `r = Recipe(profile=profile)`

Reply:
updated